test/sequential/test-child-process-execsync.js

```diff
@@ -74,11 +75,11 @@ cmd = `"${process.execPath}" -e "console.log('${msg}');"`;
 ret = execSync(cmd);
 
 assert.strictEqual(ret.length, msgBuf.length);
-assert.deepStrictEqual(ret, msgBuf, 'execSync result buffer should match');
+assert.deepStrictEqual(ret, msgBuf);
```

Comment:
The returned error message is not going to be great here and therefore it might be better to use a error message that includes both buffers. But this should not be blocking.

Reply:
@BridgeAR When I run assert.deepStrictEqual in the default behavior using foobar (as in the test) and foobaz the default assertion thrown is this:
```
AssertionError: <Buffer 66 6f 6f 62 61 7a 0a> deepStrictEqual <Buffer 66 6f 6f 62 61 72 0a>
```
Are you saying that I should change it to look something like the following?:
```
AssertionError: ret  <Buffer 66 6f 6f 62 61 7a 0a> and msgBuf <Buffer 66 6f 6f 62 61 72 0a> should be equal
```

Reply:
Na, just ignore my comment ;-)